### ISSUE TITLE
ミュートが他のブラウザを巻き込む

### ISSUE THREAD

**Issue by KimidoriBoushi3** [gre4bee/ElectronicObserver#211]
初めてwin11になった状態で報告申し上げます。
まだインストール一日目なので私だけの環境なのかどうかとかその辺は不明ですが。

この試製七四式電子観測技ENですが、他のEdge Web Viewerで動いてるブラウザも巻き込んでミュートになります。
具体的にはNicoNicoViewerというブラウザなのですが。（他のEdgeで動くブラウザに関しては不明）

で、複数の音源がある状態なので、サウンドミキサー開くとEdgeが２つあるわけなのですが、艦これ側をミュートにせずあちらだけミュートにする、という状況になりました。

あと音量0か100かなので爆音で大変なことになります。
Win10では一切見られなかった症状でした。

**Comment by myangelkamikaze** [comment]
Yeah, sound controls get problematic with more than 1 WebView2 instances running...
We might need to remove the sound manipulation option from 七四式 until it gets properly fixed in WebView2.
